setup/config_geoips

```diff
@@ -8,7 +8,8 @@ else
     CURRENT_CONFIGPATH="$( cd "$( dirname "${BASH_SOURCE[0]}")" && pwd)"
     # Used in qsub.py
     export GEOIPS_RCFILE=$CURRENT_CONFIGPATH/"$(basename "${BASH_SOURCE[0]}")"
-    export GEOIPS_VERS=`git -C $CURRENT_CONFIGPATH/.. tag | grep -v osr | tail -n 1`
+    # Sort geoips tags by creator date to grab the latest version
+    export GEOIPS_VERS=`git -C $CURRENT_CONFIGPATH/.. tag --sort=-creatordate | grep -v osr | head -n 1`
```

Comment:
Just for my information, what does this do? I read the documentation on creatordate, but I still don't understand how this change effects our geoips versioning scheme.

Reply:
It is just pulling the geoips version from the tags - it does not have any impact on actually setting the versions, etc.

Reply:
Okay, thanks!